content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -21,6 +21,11 @@ An example of the request format:
 {
   "id": 1,
   "command": "server_info"
+},
+{
+  "id": 1,
+  "command": "server_info",
+  "counters": true
 }
 ```
```

Comment:
This is not a good way to do multiple examples. People will copy the whole code block and get an error. It's OK to modify the existing example in-place—though, I would change it to `false` unless you're also going to change the example response.

Reply:
fixed in the latest tip